### PR TITLE
Release `v2021.11.0-preview`

### PR DESCRIPTION
**Pull request by andschwa**
Automated PR for new release!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+#### [vscode-powershell](https://github.com/PowerShell/vscode-powershell)
+
+- 🐛 🐢 [vscode-powershell #3652](https://github.com/PowerShell/vscode-powershell/issues/3652) - Pester code lens broken in latest preview.
+- 🐛 📟 [vscode-powershell #3653](https://github.com/PowerShell/vscode-powershell/issues/3653) - `$PROFILE` variable has changed type and behavior.
+- 🐛 📟 [vscode-powershell #3650](https://github.com/PowerShell/vscode-powershell/issues/3650) - Profiles not loading in lastest preview.
+- 🐛 📖 [vscode-powershell #3658](https://github.com/PowerShell/vscode-powershell/pull/3658) - Fix typo in settings for 'Force Clear Scrollback Buffer'. (Thanks @PrzemyslawKlys!)
+
+#### [PowerShellEditorServices](https://github.com/PowerShell/PowerShellEditorServices)
+
+- 🐛 🔍 [PowerShellEditorServices #1608](https://github.com/PowerShell/PowerShellEditorServices/pull/1611) - Improve PowerShell command and argument escaping. (Thanks @JustinGrote!)
```
Comment (StevenBucher98) [inline CHANGELOG.md:15]: Are these links correct? or do we want to link to the specific issue instead of PR?

Comment (andschwa) [reply CHANGELOG.md:15]: They're correct. The script pulls the fixed issue from the merged PR.

**Comment by SydneyhSmith** [comment]
LGTM

**Comment by andschwa** [comment]
@SydneyhSmith @StevenBucher98 If one of you could kindly click the "review" button and submit "approved" so the auto-merge goes through, that'd be much appreciated! I'm having trouble getting GitHub to let me merge (instead of squash) with my admin override rights.

**Review by StevenBucher98 — APPROVED** [review]

**Pull request merged**